.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       uses: actions/setup-go@v1
       with:
         go-version: ${{ secrets.GO_VERSION }}
-    - run: GOPROXY=direct GOSUMDB=off go get -u golang.org/x/lint/golint; go list ./... | grep -v /vendor/ | xargs -L1 /home/runner/go/bin/golint -set_exit_status
+    - run: GOPROXY=direct GOSUMDB=off go get -u golang.org/x/lint/golint; go list ./istio/... | grep -v /vendor/ | xargs -L1 /home/runner/go/bin/golint -set_exit_status
```

Comment:
I've only allowed these tests to run on valild files that are edited and not auto generated, no error will be generated for auto-generated files, as expected!

Reply:
@gunjanmimo @ashiscs have a look here and take note, if you would. This configuration could be propagated across other repos.